hellodagger/src/main/java/com/airbnb/mvrx/helloDagger/base/MvRxActivity.kt

```diff
@@ -0,0 +1,26 @@
+package com.airbnb.mvrx.helloDagger.base
+
+import android.os.Bundle
+import androidx.annotation.LayoutRes
+import androidx.appcompat.app.AppCompatActivity
+import com.airbnb.mvrx.helloDagger.appComponent
+import com.airbnb.mvrx.helloDagger.di.AssistedViewModelFactory
+import javax.inject.Inject
+
+/**
+ * Base class for Activities.
+ *
+ * This class contains an injected map of [AssistedViewModelFactory]s to make it easier to create
+ * and instantiate requested ViewModels.
+ */
+abstract class MvRxActivity(@LayoutRes layoutResId: Int = 0) : AppCompatActivity(layoutResId) {
```

Comment:
I could call this BaseActivity as to not confuse it with a class provided by the core library

Reply:
Okay, going to rename it. I think I'll go ahead and do the same with the base ViewModel class as well then.